state-sync/state-sync-v2/data-streaming-service/src/streaming_service.rs

```diff
@@ -95,7 +95,7 @@ impl<T: AptosDataClient + Send + Clone + 'static> DataStreamingService<T> {
         if let StreamRequest::TerminateStream(request) = request_message.stream_request {
             // Process the feedback request
             if let Err(error) = self.process_terminate_stream_request(&request) {
-                error!(LogSchema::new(LogEntry::HandleTerminateRequest)
+                warn!(LogSchema::new(LogEntry::HandleTerminateRequest)
                     .event(LogEvent::Error)
```

Comment:
hmm?

Reply:
this is not changing log level, this is state-sync specific structured log